main/src/com/google/refine/util/LocaleUtils.java

```diff
@@ -0,0 +1,27 @@
+package com.google.refine.util;
+
+import java.util.Locale;
+import java.util.MissingResourceException;
+import java.util.ResourceBundle;
+
+public class LocaleUtils {
+    public static void setLocale(String code) {
+        String localeCode;
+        try {
+            ResourceBundle langResource = ResourceBundle.getBundle("language_codes");
+            localeCode = (String) langResource.getObject(code);
```

Comment:
Since most of the language codes are the same in both schemes, I wonder if it would make sense to only store the mappings where it differs (suggested by @nerdyProjects who had a glimpse at my screen, thank you!), and adapt the code accordingly (if the mapping does not exist, keep the same value).

Reply:
Thanks to you and @NerdyProjects 🙏🏾 I'll do just that